python/pypto/language/distributed/op/tensor_ops.py

```
@@ -260,4 +255,48 @@ def get(
+    Returns:
+        The rebound :class:`pld.DistributedTensor` view of ``target`` —
+        identical shape / dtype / window-buffer binding, post-reduce content.
+    """
+    target_expr, signal_expr = _unwrap_distributed_tensors(
+        "pld.tensor.allreduce", target=target, signal=signal
+    )
+    call = _ir_tensor.allreduce(target_expr, signal_expr, op)
+    return DistributedTensor(expr=call)
+
+
+__all__ = ["allreduce", "alloc_window_buffer", "get", "put", "window"]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)